src/index.ts

```diff
@@ -68,5 +69,6 @@ export const release = async (
 release()
   .then(reportResults)
   .catch((error: unknown): void => {
-    setFailed(JSON.stringify(error));
+    const finalError = handleMessageOrError(error);
```

Comment:
The function name does not completely represent what the function does, maybe name it `getSetFailedErrorString` and name the file `errors.ts`? Nit: there should be an <enter> below this line.

Reply:
That's fine makes sense to rename the function. Sorry I don't understand the last point? Is it that there should be an empty line below this?

Reply:
Yess